build.gradle

```diff
@@ -131,6 +141,8 @@ test {
         excludeGroups 'cloud', 'bucket'
     }
 
+    systemProperty "dataflowRunner", System.getProperty("dataflowRunner")
```

Comment:
Could you add a comment explaining why this is needed.  I assume if you don't have it the system property doesn't get passed on to the test JVM, but it looks redundant on casual examination.